server/sonar-web/src/main/js/apps/background-tasks/store/actions.js

```diff
@@ -145,27 +144,23 @@ function fetchTasks (filters) {
     const { component } = getState();
     const parameters = mapFiltersToParameters(filters);
 
-    parameters.ps = PAGE_SIZE;
-
     if (component) {
       parameters.componentId = component.id;
     }
 
     dispatch(requestTasks(filters));
 
     return Promise.all([
-      getActivity(parameters),
-      getActivity({ ps: 1, onlyCurrents: true, status: STATUSES.FAILED }),
-      getActivity({ ps: 1, status: STATUSES.PENDING })
+      getActivity(parameters)
     ]).then(responses => {
-      const [activity, failingActivity, pendingActivity] = responses;
+      const [activity] = responses;
       const tasks = activity.tasks;
-      const total = activity.paging.total;
 
-      dispatch(receiveTasks(tasks, total));
+      dispatch(receiveTasks(tasks));
 
-      const pendingCount = pendingActivity.paging.total;
-      const failingCount = failingActivity.paging.total;
+      // FIXME request real numbers
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS1134)